### ISSUE TITLE
Sentry Python SDK does not support Apache Beam 2.33

### ISSUE THREAD

**Issue by thaddock** [getsentry/sentry-python#1231]
### Environment

self-hosted (`onpremise` deployment)

### Version

1.4.3

### Steps to Reproduce

1. Add Sentry with BeamIntegration to an Apache Beam 2.33 pipeline, for instance:
```
        sentry_sdk.init(
            dsn=os.environ.get("SENTRY_DSN"),
            release=os.environ.get("SENTRY_RELEASE", "unidentified release"),
            environment=os.environ.get("SENTRY_ENVIRONMENT", "unidentified environment"),
            integrations=[BeamIntegration()],
            before_send=before_send
        )
```
2. Run the pipeline

### Expected Result

Expected Pipeline to run successfully, and errors reported to Sentry.

### Actual Result

Received this error:
```
Traceback (most recent call last):
  (traceback from our code)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/hub.py", line 105, in _init
    client = Client(*args, **kwargs)  # type: ignore
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/client.py", line 86, in __init__
    self._init_impl()
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/client.py", line 124, in _init_impl
    "auto_enabling_integrations"
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/__init__.py", line 120, in setup_integrations
    type(integration).setup_once()
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/beam.py", line 49, in setup_once
    _wrap_inspect_call(DoFn, func_name),
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/beam.py", line 83, in _wrap_inspect_call
    from apache_beam.typehints.decorators import getfullargspec  # type: ignore
ImportError: cannot import name 'getfullargspec' from 'apache_beam.typehints.decorators' (/usr/local/lib/python3.7/site-packages/apache_beam/typehints/decorators.py)
```

**Comment by chadwhitacre** [comment]
Hey, @galuszkak! Great to see you here! 👋 😀 And thanks for the PR! ☺️

I'm a bit out of the loop on sentry-python release cadence. Hopefully @AbhiPrasad can shed some light?

**Comment by AbhiPrasad** [comment]
We are blocked by https://github.com/getsentry/craft/issues/322 + failing tests in master (though there is a fix at https://github.com/getsentry/sentry-python/pull/1212). The moment that those two get resolved, we will cut a release and let you know.

**Comment by chadwhitacre** [comment]
Thanks @AbhiPrasad. Looks like you are on #1212. 👍 https://github.com/getsentry/craft/issues/322 looks like the bigger issue ...

**Comment by AbhiPrasad** [comment]
Fix released as part of SDK version [`1.5.0`](https://github.com/getsentry/sentry-python/releases/tag/1.5.0)
